.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!--
+Thanks for filing an issue! Before hitting the button, give this a read.
+
+If you are reporting a new issue, make sure that we do not have any duplicates already open. You can ensure this by searching the issue list for this repository. If there is a duplicate, please close your issue and add a comment to the existing issue instead.
+
+If you suspect your issue is a bug, please add as much context as you can using the template below. If it is not a bug, remove any sections you don't need and explain how we can help.
+
+As a basic rule, if you cannot provide enough information to continue addressing the issue within 7 days, a maintainer will close it. We will, however, reopen it if you later provide the information. Thanks again.
+-->
+
+**Snap version** (use `snapctl -v`):
```

Comment:
We have an open issue SDI-2242 to address this across all plugins. For now I'm using the output of the sync tool and not manually changing it. Someone also pointed out this should be snapteld -v since we typically don't care what client version you use.